gymnasium/envs/registration.py

```diff
@@ -567,7 +567,7 @@ def register(
     disable_env_checker: bool = False,
     additional_wrappers: tuple[WrapperSpec, ...] = (),
     vector_entry_point: VectorEnvCreator | str | None = None,
-    **kwargs: Any,
+    kwargs: dict = {},
```

Comment:
I know that pycharm shouts at me if I do this as `{}` is mutable object. 
Could you change to `dict | None = None`

Reply:
True, done!